index.html

```diff
@@ -4,33 +4,41 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Moody - Where people become happy</title>
+    <link rel="stylesheet" href="style.css">
```

Comment:
Keep this line below the script tag so that tailwind styles get rendered first before this as top-to-bottom rule is followed by browsers.

Reply:
> Keep this line below the script tag so that tailwind styles get rendered first before this as top-to-bottom rule is followed by browsers.

done